### ISSUE TITLE
Modifying spectral surface albedo

### ISSUE THREAD

**Issue by jtolento** [AER-RC/RRTMG_SW#3]
I'm running rrtmg_sw on the example file input_rrtm_SAW-clr. I would expect once I change the file from broadband emissivity of 0.8 to spectral emissivity with 0.8 for all bands, the vertical profile should remain the same. But they don't. The modified file outputs different results for variables such as the daily heating rate. I've attached a copy of the input file that I modified.  
I would appreciate it if this can be cleared up, as to whether or not I have an incorrect understanding on how the model works, as I aim to make future changes to implement a spectrally resolved albedo.
From my understanding, looking at the source code, to input a spectrally resolved albedo, I input the emissivity as (1-albedo) into each of the 14 bands in the input file. Is this correct? or am I missing something? Having input an array of values for emissivity, rrtm is unable to return the verticle profile of the upwards and net fluxes. Furthermore, it also doesn't return values for the heating rates in the lower part of the atmosphere.
 
I would appreciate any tips or resources that can point me in the right direction to resolve these issues. 
Thank you,
-Juan Tolento

[input_rrtm_SAW_spec8.txt](https://github.com/AER-RC/RRTMG_SW/files/6380227/input_rrtm_SAW_spec8.txt

**Comment by mjiacono** [comment]
Juan: Your modification to the input file to set the spectral emissivity values in all 14 bands (IEMIS=2) is correct except for one important point. The format of the spectral emissivity values must be 14F5.3. If you make this change and use the same value in all 14 bands, the code will reproduce the same result that you get when you use the single, spectrally constant emissivity value (IEMIS=1). See the file doc/rrtmg_sw_instructions.txt for more information about the input file settings.

**Comment by jtolento** [comment]
Thank you for getting back to me
Originally I used they 14F5.3 in the input files (as specified in the source code) format, using 0.800 for all 14 bands and found that the daily heating rates produces a difference of ~0.2 degrees through out the vertical profile when compared to the broadband of 0.8 . I have since modified the source code (file rrtmg_sw.1col.f90) to read in 9012 as '9012 format (11x,i1,2x,i1,14f7.5)' to take in the input file I originally posted with the spectral albedo set to 0.80000 for all bands. This is the only change I have made to the source code. The result ended up with significantly closer to the broadband output, but still off by ~0.05 degrees heating/day when comparing spectral vs broadband.
